myapp/app/models/user.rb

```diff
@@ -21,5 +21,22 @@ class User < ApplicationRecord
 
   has_many :tasks, dependent: :delete_all
 
+  before_destroy :stop_destroy
+
   has_secure_password
+
+  scope :admin, -> { where(is_admin: true) }
+
+  def the_only_admin?
+    return false unless is_admin
+    return false if User.admin.unscope(:order).limit(2).count == 2
```

Comment:
```suggestion
    return false if User.admin.limit(2).count == 2
```

do we need `unscope`? 
I tried on rails console and found no difference without it.

```Ruby
irb(main):007:0> User.admin.unscope(:order).limit(2).count
   (0.7ms)  SELECT COUNT(*) FROM (SELECT 1 AS one FROM `users` WHERE `users`.`is_admin` = TRUE LIMIT 2) subquery_for_count
=> 1
irb(main):008:0> User.admin.limit(2).count
   (1.0ms)  SELECT COUNT(*) FROM (SELECT 1 AS one FROM `users` WHERE `users`.`is_admin` = TRUE LIMIT 2) subquery_for_count
=> 1
```

Reply:
yeah, it can be removed now. Just my habits to add this when doing query with order does not matter. Removed it now 852b45ebc68032a3b1d28eb2761cfb67379b7c3c ~